src/index.js

```diff
@@ -17,62 +17,72 @@
  * limitations under the License.
  */
 
-const request = require('request');
-const requestPromise = require('request-promise-native');
-const AccessControl = require('./accessControl');
-const Acsrf = require('./acsrf');
-const AjaxSpider = require('./ajaxSpider');
-const Alert = require('./alert')
-const AlertFilter = require('./alertFilter');
-const Ascan = require('./ascan');
-const Authentication = require('./authentication');
-const Authorization = require('./authorization');
-const Automation = require('./automation');
-const Autoupdate = require('./autoupdate');
-const Brk = require('./brk');
-const Context = require('./context');
-const Core = require('./core');
-const Exim = require('./exim');
-const ForcedUser = require('./forcedUser');
-const Graphql = require('./graphql');
-const HttpSessions = require('./httpSessions');
-const Network = require('./network');
-const Openapi = require('./openapi');
-const Params = require('./params');
-const Pnh = require('./pnh');
-const Pscan = require('./pscan');
-const Reports = require('./reports');
-const Replacer = require('./replacer');
-const Reveal = require('./reveal');
-const Retest = require('./retest');
-const Revisit = require('./revisit');
-const RuleConfig = require('./ruleConfig')
-const Script = require('./script');
-const Search = require('./search');
-const Selenium = require('./selenium');
-const SessionManagement = require('./sessionManagement');
-const Soap = require('./soap');
-const Spider = require('./spider');
-const Stats = require('./stats');
-const Users = require('./users');
-const Wappalyzer = require('./wappalyzer');
-const Websocket = require('./websocket');
+
+// const request = require("request");
+// const requestPromise = require("request-promise-native");
+const axios = require("axios");
+const AccessControl = require("./accessControl");
```

Comment:
These changes should not be done in this PR.

Reply:
Sorry, I didn't get the point, aren't we supposed to remove the deprecated library?

Reply:
Th mentioned line is just changing the quotes.

Reply:
<img width="829" alt="image" src="https://user-images.githubusercontent.com/47940898/219375520-9ccd7aae-9a61-4785-8a88-7d3a5a4327a5.png">
Out of sight, out of mind :| I cant see the quotes difference.

Reply:
That's because of mix of changes being done, the previous `accessControl` line https://github.com/zaproxy/zap-api-nodejs/pull/29/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556L22 which was using single quotes.

Reply:
ah, I see, maybe linter did that. Okay I will fix it.

Reply:
It was prettier extension doing that.